### PR TITLE
Fixed (301) permanent redirect, wrong file type on node-webkit download.

### PR DESCRIPTION
**Pull request by Chao-Man**
- Changed URL.

### CODE REVIEW

File: download/index.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ module.exports = yeoman.generators.Base.extend({
     this.downloadNodeWebkit = true;
   },
   _getDownloadUrl: function () {
-    return 'http://dl.nwjs.io/' + this.nodeWebkitVersion + '/node-webkit-' + this.nodeWebkitVersion + '-';
+    return 'http://dl.node-webkit.org/' + this.nodeWebkitVersion + '/node-webkit-' + this.nodeWebkitVersion + '-';
   },
   _getDownloadTmpUrl: function (version) {
-    return 'http://dl.nwjs.io/' + version + '/node-webkit-' + version + '-linux-x64.tar.gz';
+    return 'http://dl.node-webkit.org/' + version + '/node-webkit-' + version + '-linux-x64.tar.gz';
```
Comment (mschaaf) [inline download/index.js]: Why this change? http://dl.node-webkit.org redirects to http://dl.nwjs.io/

**Comment by Chao-Man** [comment]
I was receiving an "Wrong content type" error upon downloading node-webkit. The issue seemed to be mitigated after changing the URL to  http://dl.node-webkit.org. However, I can't seem to replicate the issue again. Apologies.

**Comment by mschaaf** [comment]
We plan a release with a fix to follow redirects. Maybe this will solve this too.
